superset/assets/javascripts/dashboard/v2/components/DashboardBuilder.jsx

```diff
@@ -105,15 +108,19 @@ class DashboardBuilder extends React.Component {
               index={0}
               renderTabContent={false}
               onChangeTab={this.handleChangeTab}
+              cells={this.props.cells}
```

Comment:
why does `DashboardBuilder` need `cells`? can you just pass directly via the `DashboardComponent` container/redux connection?